README.md

```diff
@@ -45,32 +27,56 @@ For DWI:
 
 Specify the path of preprocessed dataset with the flag `-path-data`. 
 
-#### Launch processing
+### Launch processing
 
 ```
 sct_run_batch -jobs -1 -path-data <PATH_DATA> -path-output <PATH-OUTPUT> -script process_data.sh -script-args exclude_reg.yml
 ```
 
+A `process_data_clean` folder is created in <PATH-OUTPUT> where the cropped data and derivatives are included. Here, only the images that have a manual segmentation and soft segmentation are transfered.
+
+### Quality control
+
+After running the analysis, check your Quality Control (qc) report by opening the file <PATH-OUTPUT>/qc/index.html. Use the "search" feature of the QC report to quickly jump to segmentations or labeling issues.
+
+**1. Segmentations**
+
+If segmentation issues are noticed while checking the quality report, proceed to manual correction using the procedure below:
+
+* In QC report, search for "deepseg" to only display results of spinal cord segmentation
+* Review segmentation and spinal cord.
+* Click on the F key to indicate if the segmentation is OK ✅, needs manual correction ❌ or if the data is not usable ⚠️ (artifact). Two .yml lists, one for manual corrections and one for the unusable data, will automatically be generated.
+* Download the lists by clicking on **Download QC Fails*** and on **Download Qc Artifacts**.
+
+Proceed to manual correction using FSLeyes or ITK snap. Uplaod the manual segmentations (_seg-manual.nii.gz) with json sidecar in the derivatives.
+Re-run the analysis: [Launch processing](#launch-processing)
+
+**2. Registrations**
+
+* In QC report, search for "sct_register_multimodal" to only display results of registration.
+* Click on the F key to indicate if the registration is OK ✅, needs to be excluded ❌ or if the data is not usable ⚠️ (artifact). Two .yml lists, will automatically be generated.
+* Downlaod the list by clicking on **Download QC Fails** and add the file names under `FILES_REG` to https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/blob/main/exclude_reg.yml
+* Re-run the analysis: [Launch processing](#launch-processing)
+
+**3. Soft segmentations**
+
+* In QC report, search for "softseg" to only display results of spinal cord soft segmentation
+* Click on the F key to indicate if the soft segmentation is OK ✅, needs manual correction ❌ or if the data is not usable ⚠️ (artifact). Two .yml lists, one for manual corrections and one for the unusable data, will automatically be generated.
+* Downlaod the list by clicking on **Download QC Fails**.
+* Uplaod the soft segmentations under `data-multi-subject/darivatives/labels_softseg`.
```

Comment:
```suggestion
* Upload the soft segmentations under `data-multi-subject/derivatives/labels_softseg`.
```

write 100 times 'upload/download' on the black board 😉

Reply:
it was more of a copy/paste error 😂